src/ServiceControl.Audit/ServiceControl.Audit.csproj

```diff
@@ -7,6 +7,10 @@
     <ApplicationIcon>Operations.ico</ApplicationIcon>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <AppConfig>App.$(Configuration).config</AppConfig>
```

Comment:
I'd just move this up into the `PropertyGroup` above instead of defining a separate one just for this property.